ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -166,21 +166,12 @@ then
 			fi
 		fi
 
-		#
-		# Set disarmed values for aux out.
-		# Transitional support until all configs are updated.
-		#
-		if [ $PWM_ACHDIS == none ]
-		then
-			set PWM_ACHDIS ${PWM_AUX_OUT}
-		fi
-
 		#
 		# Set disarmed, min and max PWM_AUX values.
 		#
-		if [ $PWM_AUX_DISARMED != none -a $PWM_ACHDIS != none ]
+		if [ $PWM_AUX_DISARMED != none -a $PWM_AUX_OUT != none ]
```

Comment:
This block can now be removed, because the `disarmed` values will be overwritten by the next block.

Reply:
Done. Thanks!